repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/Recover.scala

```diff
@@ -0,0 +1,65 @@
+package com.snowplowanalytics.snowplow.storage.bigquery.repeater
+
+import fs2.Stream
+import cats.effect.Concurrent
+import cats.effect.Timer
+import blobstore.Path
+import blobstore.implicits.GetOps
+import io.circe.JsonObject
+import java.time.Instant
+import java.util.UUID
+import io.circe.generic.auto._, io.circe.parser._
+import cats.effect.Sync
+import cats.syntax.all._
+
+object Recover {
+
+  //TODO: hardcode proper GCS path
+  val DeadQueuePath = Path("gs://sp-storage-loader-failed-inserts-dev1-com_snplow_eng_gcp/dead_queue")
+
+  def recoverFailedInserts[F[_]: Timer: Concurrent](resources: Resources[F]): Stream[F, Unit] =
+    resources
+      .store
+      .list(DeadQueuePath)
```

Comment:
Do we have one `Path` for each batch of failed inserts or is it always the same? In later case if failed inserts keep being added we will always try to reprocess them all. I didn't see a place where we remove a successfully recovered file.

Reply:
Good point. It's not removed at the moment, I'll probably ask support to remove it after successful recovery. To not read the same file twice the filtering part (in TODO) will ensure we only recover the folder once. 
I don't think there will be problems like every second event will fail.
I rather assume that all will be ok, or fail (if there is a mistake in the recovery).

Reply:
> To not read the same file twice the filtering part (in TODO) will ensure we only recover the folder once.

So you want to maintain a manifest outside of BQ loader ?

Reply:
No. I will simply filter for proper dates in which the failed events occurred. I will not do it for all past and current events in dead_queue, only for particular dates.